File: BNOI - Group E/Piknik/Piknik.cs
using System;

class Piknik
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split(' ');
        float n = (int.Parse(input[0]) * 0.4f) / float.Parse(input[1] + "." + input[2]);
        Console.WriteLine((n % 1 > 0) ? (int)(n) + 1 : (int)(n));
    }
}

File: BNOI - Group E/Piknik/Piknik.js
function solve(args) {
    var input = args[0].split(' ');
    var n = (parseInt(input[0]) * 0.4) / parseFloat(input[1] + '.' + input[2]);

    return (n % 1 > 0) ? parseInt(n) + 1 : parseInt(n);
}
